srcpkgs/mutter/patches/meta-renderer-x11.patch

```diff
@@ -1,8 +1,8 @@
---- src/backends/x11/meta-renderer-x11.c	2017-05-11 06:30:09.000000000 -0600
-+++ src/backends/x11/meta-renderer-x11.c	2017-06-05 12:14:08.090473267 -0600
-@@ -79,14 +79,6 @@
-   cogl_renderer_set_custom_winsys (cogl_renderer, get_x11_cogl_winsys_vtable);
+--- src/backends/x11/meta-renderer-x11.c	2019-03-11 16:01:02.000000000 -0700
++++ src/backends/x11/meta-renderer-x11.c	2019-03-11 23:44:53.274968331 -0700
+@@ -85,14 +85,6 @@
    cogl_xlib_renderer_set_foreign_display (cogl_renderer, xdisplay);
+   cogl_xlib_renderer_request_reset_on_video_memory_purge (cogl_renderer, TRUE);
  
 -  /* Set up things so that if the INTEL_swap_event extension is not present,
```

Comment:
Also, just out of curiosity, why is this patched out?

Reply:
I think this was about some weird lags on Intel? Dunno and sadly there's no patch header :/